TestModels/SharedMakefile.mk

```diff
@@ -185,28 +246,31 @@ transpile_dependencies:
 # StandardLibrary is filtered out from dependent-model patsubst list;
 #   Its model is contained in $(LIBRARY_ROOT)/model, not $(LIBRARY_ROOT)/../StandardLibrary/Model.
 _polymorph:
-	@: $(if ${CODEGEN_CLI_ROOT},,$(error You must pass the path CODEGEN_CLI_ROOT: CODEGEN_CLI_ROOT=/path/to/smithy-dafny/codegen/smithy-dafny-codegen-cli));
 	cd $(CODEGEN_CLI_ROOT); \
-	$(GRADLEW) run --args="\
-	$(DAFNY_VERSION_OPTION) \
+	./../gradlew run --args="\
+	--dafny-version $(DAFNY_VERSION) \
```

Comment:
We have already defined `GRADLEW` above. 
Should we not use it?

Reply:
`GRADLEW` previously meant "what gradlew should we use to run the polymorph CLI", but I'm using it in the unified makefile to mean "what gradlew should we use to build/test/deploy/etc. each `runtimes/java` project.

I did that because the former doesn't really need to be customized AFAICT - it makes sense to always use the `gradlew` sitting next to the polymorph code under `codegen`. But the latter definitely does need be configurable with the current project layouts, because the `TestModels/**/runtimes/java` folders don't have their own gradlew, whereas the projects in the MPL/DB ESDK/ESDK do and are currently building with them.

I should at least add a comment clarifying this to the definition of `GRADLEW`. But you might have a stronger opinion that I'm missing a use case for the current definition of it, or that we should require polymorph libraries to include a copy of gradlew.

Reply:
I do not have an opinion.
In theory, it would be optimal for everyone to use the same Gradlew, 
as that might cut down on Gradle Deamon boot ups,
but that theory is pretty shaky.